pipeline-model-definition/src/test/resources/errors/additionalInvalidExpressionsInEnvironment.groovy

```diff
@@ -22,24 +22,30 @@
  * THE SOFTWARE.
  */
 
+pipeline {
+    environment {
+        FOO =
+            "B" +
+                "A" -
+                "R"
+        BAR = 1 + credentials("foo")
+        _UNDERSCORE = "VALID" + "FOO" ?: "BAR"
```

Comment:
This is literally just here to blow up on the `?:` elvis expression - the content is kinda irrelevant.